tests/core/fs/workspacefs/test_workspace_fs.py

```diff
@@ -257,10 +257,15 @@
     assert await alice_workspace.read_bytes("/foo/bar") == b"abcde"
 
     assert await alice_workspace.write_bytes("/foo/bar", b"xyz", offset=1)
-    assert await alice_workspace.read_bytes("/foo/bar") == b"axyze"
-
+    assert await alice_workspace.read_bytes("/foo/bar") == b"axyz"
     assert await alice_workspace.write_bytes("/foo/bar", b"[append]", offset=-1)
-    assert await alice_workspace.read_bytes("/foo/bar") == b"axyze[append]"
+    assert await alice_workspace.read_bytes("/foo/bar") == b"append]"
```

Comment:
The expected value is `b"axyze[append]"` (see my previous comment about negative offset)

Reply:
C'est fixé.